pages/requests/[id].js

```diff
@@ -95,8 +103,8 @@ const Request = () => {
           <Title title={request.title} />
           <CollapsibleSection header='Additional Information' description={request.htmlDescription} />
           <Title addClass='mt-4' title='Documents' size='small' />
-          {documents.length ? documents.map(document => (
-            <Document key={request.id} document={document} addClass='mt-3' />
+          {documents.length ? documents.map((document, index) => (
+            <Document key={index} document={document} addClass='mt-3' />
```

Comment:
```suggestion
            <Document key={`${request.id}-${index}`} document={document} addClass='mt-3' />
```

Reply:
perhaps something like the above instead? it would resolve the duplicate key problem for when there are 2+ documents. it would also provide more specificity than the index alone.